src/main/java/io/jmix/uisamples/view/flowui/pivottable/showaction/PivotTableShowAction.java

```diff
@@ -0,0 +1,54 @@
+package io.jmix.uisamples.view.flowui.pivottable.showaction;
+
+import io.jmix.core.Messages;
+import io.jmix.flowui.Actions;
+import io.jmix.flowui.component.grid.DataGrid;
+import io.jmix.flowui.kit.component.button.JmixButton;
+import io.jmix.flowui.view.*;
+import io.jmix.pivottableflowui.action.PivotTableViewBuilder;
+import io.jmix.pivottableflowui.action.ShowPivotTableAction;
+import io.jmix.pivottableflowui.kit.component.model.Renderer;
+import io.jmix.pivottableflowui.kit.component.model.Renderers;
+import io.jmix.uisamples.entity.TipInfo;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Arrays;
+import java.util.List;
+
+@ViewController("pivottable-show-action")
+@ViewDescriptor("pivottable-show-action.xml")
+public class PivotTableShowAction extends StandardView {
```

Comment:
In this example, another view opens, the `UI Samples` menu and controls disappears. This will definitely scare users. 
Can we open this view in a dialog mode?

Reply:
This example will be excluded pending a fix in the framework